operators/argo-controller/test-integration-requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Pull from github main branch until >0.8.2 is available in pypi
+# pytest-operator  # integration only.  Could be in separate requirements.txt
+https://github.com/charmed-kubernetes/pytest-operator/archive/refs/heads/main.zip
```

Comment:
0.8.2 seems to be out, do you need a newer release?

Reply:
Switched back to pypi now that 0.8.3 is out